src/routes/blog/index.svelte

```diff
@@ -193,6 +193,12 @@
           }
         }
 
+        a > h2 {
+          overflow: hidden;
+          text-overflow: ellipsis;
+          min-width: 0;
```

Comment:
unless what you meant to do was a max width/height as this does not currently truncate
![image](https://user-images.githubusercontent.com/8981287/191657344-1b181a39-46f3-41ae-ac18-562713d5b0e0.png)

Reply:
I yoinked it out